In [3]:
%pip install seaborn

Note: you may need to restart the kernel to use updated packages.


In [5]:
import seaborn as sns



In [7]:
titanic = sns.load_dataset('titanic') 

In [11]:
titanic.head()

,survived,pclass,sex,age,sibsp,parch,fare,embarked,class,who,adult_male,deck,embark_town,alive,alone
0,0,3,male,22.0,1,0,7.2500,S,Third,man,True,NaN,Southampton,no,False
1,1,1,female,38.0,1,0,71.2833,C,First,woman,False,C,Cherbourg,yes,False
2,1,3,female,26.0,0,0,7.9250,S,Third,woman,False,NaN,Southampton,yes,True
3,1,1,female,35.0,1,0,53.1000,S,First,woman,False,C,Southampton,yes,False
4,0,3,male,35.0,0,0,8.0500,S,Third,man,True,NaN,Southampton,no,True


In [6]:
import pandas as pd


In [9]:
titanic.shape

(891, 15)

In [10]:
titanic.dtypes

survived          int64
pclass            int64
sex                 str
age             float64
sibsp             int64
parch             int64
fare            float64
embarked            str
class          category
who                 str
adult_male         bool
deck           category
embark_town         str
alive               str
alone              bool
dtype: object

In [12]:
titanic.isna().sum()

survived         0
pclass           0
sex              0
age            177
sibsp            0
parch            0
fare             0
embarked         2
class            0
who              0
adult_male       0
deck           688
embark_town      2
alive            0
alone            0
dtype: int64

In [13]:
titanic["class"].value_counts()

class
Third     491
First     216
Second    184
Name: count, dtype: int64

remaque_01 : le type de la colonne age est mal choisi
remarque_02 : plusieurs colonnes n'ont pas lieu d'être (trop de rédondances)
remarque_03 : la colonne deck est presque vide (688 valeurs manquantes pour 891 lignes)